apps/api/src/app/events/services/workflow.queue.service.ts

```diff
@@ -64,67 +58,24 @@ export class WorkflowQueueService {
   }
 
   public async work(job: IJobEntityExtended) {
```

Comment:
This can be made private when we are using the usecase in e2e tests instead of this method